examples/mockups/product.html

```diff
@@ -0,0 +1,466 @@
+              </p>
+              
+              <div class="ui relaxed divided list">
+
+                <div class="item">
+                  <i class="icon check circle green"></i>
+                  <div class="content">
+                    <div class="description">Always up to date</div>
+                  </div>
+                </div>
+                
+                <div class="item">
+                  <i class="icon check circle green"></i>
+                  <div class="content">
+                    <div class="description">Automatic deploy</div>
+                  </div>
+                </div>
+                
+                <div class="item">
+                  <i class="icon check circle green"></i>
+                  <div class="content">
+                    <div class="description">Multiple Versions</div>
+                  </div>
+                </div>
+
+              </div>
+
+            </div>
+            <div>
+              <a href="" class="ui bui button basic inverted fluid large">How RTD works for teams</a>
+            </div>
+          </div>
+
+        </div>
+      </div>
+
+    </section>
+
+    <!-- Features -->
+    <section class="ui vertical segment light padded" id="features">
+      <div class="ui container">
+        <h2 class="ui header">Features</h2>
+
+        <div class="ui two stackable cards">
+
+          <div class="card">
+            <div class="content">
+              <h4>Webhooks</h4>
+              <p>
+                Whenever you push code to your favorite version control system,
+                whether that is Git, Mercurial, Bazaar, or Subversion, we will
+                automatically build your docs so your code and documentation are
+                never out of sync.
+              </p>
+            </div>
+            <div class="extra content">
+              <div class="ui labels">
+                <span class="ui label basic large"><i class="icon primary github"></i> GitHub</span>
+                <span class="ui label basic large"><i class="icon primary gitlab"></i> GitLab</span>
+                <span class="ui label basic large"><i class="icon primary bitbucket"></i> BitBucket</span>
+              </div>
+            </div>
+          </div>
+
+          <div class="card">
+            <div class="content">
+              <h4>Multiple formats</h4>
+              <p>
+                Of course we build and host your docs for the web, but they are
+                also vieweable as PDFs, as single page HTML, and for eReaders.
+                No additional configuration is required.
+              </p>
+            </div>
+            <div class="extra content">
+              <div class="ui labels">
+                <span class="ui label basic large"><i class="icon primary code"></i> html</span>
+                <span class="ui label basic large"><i class="icon primary file alternate outline"></i> pdf</span>
+              </div>
+            </div>
+          </div>
```

Comment:
What do you think about adding space to these cards to show a graphic element -- ie a minimal terminal/browser graphic illustrating the workflow? Cards can have image headers, or in the body.

This seems like it would probably be a follow up project to this page and we should open an issue to track. We probably don't yet want to get bogged down on producing graphics for all of the features -- and we'd have to decide if we are producing graphics for all features/steps, or just a select couple, or what that really looks like when cards in a row have vastly differing heights due to graphics.

Reply:
> What do you think about adding space to these cards to show a graphic element -- ie a minimal terminal/browser graphic illustrating the workflow?

I'm all in favour of adding more illustrative graphic to help us explain. We do have the obvious problem of producing these graphics... Up to a point I think it's helpful to have a more generic graphic that we can find on some image or icons pack to try illustrating ideas. If we start adding graphics to every block then at some point these generic images start becoming more noise than help and we would benefit with images produced specifically to explain our concepts.

> Cards can have image headers, or in the body.

Yes, images are one of the main elements in cards actually: https://semantic-ui.com/views/card.html

> This seems like it would probably be a follow up project

I agree. My idea for now, so we can keep moving forward defining our messaging, would be to use the more generic images where we can for now and produce just a very small number of images at this point - the infographic being one of those. Later we can come back and produce or modify images that might seem a bit out of place.

Reply:
> we start adding graphics to every block however, at some point these generic images become noise

Yeah absolutely. I was picturing not all (probably most?) blocks would not have graphics. I definitely agreed that would be too much effort to maintain currently. I do see that only using graphics on some blocks could feel awkward however, given the two column flex layout -- unless we put two graphical features in a row, that would work well.

> Later we can come back and produce or modify images that might seem a bit out of place.

Agreed. Let's add an issue and come back to this one.

Reply:
https://github.com/readthedocs/site-community/issues/8